tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -110,6 +110,10 @@ spec:
       type: string
       description: Run policy checks with the provided time.
       default: "now"
+    - name: EXTRA_RULE_DATA
+      type: string
+      description: Merge additional Rego variables into the policy data. Use syntax "key=value,key2=value2..."
+      default: "key=value"
```

Comment:
Probably needs to be blank by default, which would solve the snapshot failures in the acceptance tests. But I think maybe there'll be some extra work to avoid giving ec `--extra-rule-data=` with no values where there are no values.

Reply:
Per offline discussion, changed this to empty string. Should still work, and will fix acceptance tests.